zipline/data/minute_bars.py

```diff
@@ -265,7 +274,8 @@ def read(cls, rootdir):
 
     def __init__(
         self,
-        ohlc_ratio,
+        default_ohlc_ratio,
```

Comment:
Not 100% sure we need to put the prefix of `default` here, but not going to block on it.